talisker/gunicorn.py

```diff
@@ -61,12 +61,11 @@ class GunicornLogger(Logger):
     """Custom gunicorn logger to use structured logging."""
 
     def get_extra(self, resp, req, environ, request_time):
-
-        msg = "%s %s" % (environ['REQUEST_METHOD'], environ['RAW_URI'])
-
         status = resp.status
         if isinstance(status, (str, bytes)):
             status = status[:3]
+        else:
+            status = str(status)
```

Comment:
Yeah, I just realised the it was relying on upstream str() conversion to work, so I slipped it in.
Bizarrely, in some circumstances, status is an int, but I've not been able to get to the bottom of why.